content/docs/user-guide/visualization/animation/animation-editor/ragdoll/ragdoll-physics-setup.md

```diff
@@ -0,0 +1,145 @@
+
+The **Twist Limit** describes rotations around an axis rigidly attached to the child. That axis can be orientated arbitarily relative to the child during the setup of the joint limit, but it is usually easiest to visualize if it points directly along the child (from the elbow to the wrist in the example). If you imagine holding your arm out straight in front of you, you can rotate around the forearm so that your palm is face up or face down, but it is uncomfortable to rotate much further in either direction, i.e. there is a limit to how much you can twist around that axis. If you were holding a pen between your fingers at right angles to your palm, then the pen would describe a fan shape as you rotate your forearm.
+
+The **Swing Limit** describes how the axis attached to the child can move relative to the parent. Continuing with the elbow example, if you consider holding your upper arm still, you can bend your elbow so that your hand touches your shoulder, or extend it so that the arm is straight. There is also a range of motion to the left or right (think about arm wrestling). Overall, you can imagine a cone rigidly attached to the upper arm which describes the full range of possible orientations for the forearm. Note that because the range of motion of the forearm is not symmetrical (you can bend your elbow more in some directions than in others), the cone needs to be rotated relative to the direction of the upper arm. 
+
+Putting all this together, there are several parameters which can be configured for each joint limit:
++ The axis attached to the child body of the joint, which is rendered as a long arrow. It is usually easiest to align this to point along the child body, and joint limits will automatically be configured that way when they are first created. 
++ The orientation of the **Swing Limit**, which is rendered as a cone, relative to the parent body.
++ The angular extents of the **Swing Limit** cone.
++ The minimum and maximum values for the **Twist Limit**, which is rendered as a fan shape.
+
+If a single joint is selected, these parameters can be modified using manipulators accessible through a **Viewport UI Cluster** at the top left of the **Atom Render Window** (below the collider cluster if it is available). Different manipulator modes can be selected by clicking on the following icons in the **Viewport UI Cluster**.
+
+| Mode | Icon | Description
+| - | - | - |
+| **Parent local rotation** | | Activates manipulators allowing the rotation of the joint parent frame to be modified. This allows the cone for the swing limit cone and the arc for the twist limit to be orientated relative to the frame of the joint's parent. |
+| **Child local rotation** | | Activates manipulators allowing the rotation of the joint child frame to be modified. This allows the axis which moves around inside the joint limit to be orientated relative to the joint's frame. |
```

Comment:
Any icons for these, or the last one in this table?

Reply:
Thanks for reminding me. They were waiting on UX to finalize, but that's done now, I will update shortly.

Reply:
Added in the most recent commit.